Breeze.BreezeServer/Program.cs

```diff
@@ -115,10 +115,21 @@ public static void Main(string[] args)
                 logger.LogInformation("{Time} Node registration has already been performed", DateTime.Now);
             }
 
+            // Perform collateral balance check and report the result
+            Money missingFunds;
+            if (registration.VerifyCollateral(config, out missingFunds))
+            {
+                logger.LogInformation($"{{Time}} The collateral address {config.TumblerEcdsaKeyAddress} has sufficient funds.", DateTime.Now);
+            }
+            else
+            {
+                logger.LogWarning($"{{Time}} The collateral address {config.TumblerEcdsaKeyAddress} doesn't have enough funds. Collateral requirement is {RegistrationParameters.MASTERNODE_COLLATERAL_THRESHOLD} but only {missingFunds} is available at the collateral address. This is expected if you have only just run the masternode for the first time. Please send funds to the collateral address no later than {RegistrationParameters.WINDOW_PERIOD_BLOCK_COUNT} blocks after the registration transaction.", DateTime.Now);

```

Comment:
missingFunds seems like a bit of a misleading variable name

Reply:
After discussion with @ferdeen I have change it to `collateralShortfall`